packages/fast-components-react-base/src/image/README.MD

```diff
@@ -0,0 +1,31 @@
+# Implementation
+
+## Implementation example
+```javascript
+
+<!-- Returns <img />-->
+<Image 
+    src='https://placehold.it/539x300/2F2F2F/171717'
+    alt='Placeholder with grey background and dimension watermark without any imagery'
+/>
+
+<!-- Returns <img /> using SrcSet-->
+<Image 
+    src='https://placehold.it/539x300/2F2F2F/171717'
+    srcSet='https://placehold.it/1399x600/2F2F2F/171717 1399w, https://placehold.it/1083x500/2F2F2F/171717 1083w, https://placehold.it/767x400/2F2F2F/171717 767w'

```

Comment:
img element does not use srcset, just src. see: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement

Reply:
HTMLImageElement.srcset is listed on the site you just sent me.

Reply:
Sure, but its got the little icon for experimental, which means it isn't supported across the board

Reply:
Carry over from the other thread; we can surface these but they should be optional and not have defaults.